app/serializers/user_serializer.rb

```diff
@@ -20,4 +20,8 @@ def wallet_balance
       object.wallet_balance
     end
   end
+
+  def total_referral_rewards
+    Referral.rewarded.for_referrer(object).sum(&:reward)
```

Comment:
Using `sum(&:reward)` would bring all the records data in memory and then do the sum.
I think triggering a SQL SUM (by `sum(:reward)`) might be better in performance.